backendImpl/src/main/java/com/virtuslab/gitmachete/backend/impl/GitMacheteRootBranch.java

```diff
@@ -25,6 +29,11 @@ public GitMacheteRootBranch(String name, List<GitMacheteNonRootBranch> downstrea
       IGitMacheteCommit pointedCommit,
       ISyncToRemoteStatus syncToRemoteStatus,
       @Nullable String customAnnotation) {
+    LOG.debug(
+        () -> "Creating GitMacheteRootBranch(name = ${name}, " +
```

Comment:
Hmmm :thinking: What do you think about describing logs in imperative mood (like commits)? I saw `set` (instead of `setting`?) before so I assume there is no convention yet (probably it's unnecessary). I am interested in @PawelLipski's view.

Reply:
Hmm it's better if each log implies whether the action *has been done* or *is about to be done* or *is currently ongoing*. So while in case of commit message I'm favor of infinitive, for logs it's IMO better to be more specific wrt. grammar form.